App.xaml

```diff
@@ -0,0 +1,10 @@
+﻿<Application x:Class="SimpleMvvmSamples.WPF.App"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:locators="clr-namespace:SimpleMvvmSamples.WPF.Locators"

```

Comment:
You can remove the use of Locators, since we should use dependency injection instead.